### DISCUSSION TITLE
Getting loading chunk error on performing any actions

### DISCUSSION THREAD

**Discussion by vishakha-vijay-bh** [excalidraw/excalidraw#5145] [Q&A]
![MicrosoftTeams-image](https://user-images.githubusercontent.com/88440957/166922410-c152fc58-8a5a-42f0-918c-cc1744920b32.png)

**Comment by ad1992** [comment]
@vishakha-vijay-bh which package and version are you using?

**Reply by vishakha-vijay-bh** [reply]
@ad1992  "@excalidraw/excalidraw": "0.10.0"

**Reply by ad1992** [reply]
looks like you are serving the assets from a different path. Did you update `window.EXCALIDRAW_ASSET_PATH` and also copied the assets to that path ?

**Reply by vishakha-vijay-bh** [reply]
@ad1992 can you pls share the reference to this asset path

**Reply by ad1992** [reply]
Sure, can you please go through the [installation](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#installation), you will need to copy the assets folder to the the path from where you want to serve as mentioned in the docs

**Reply by vishakha-vijay-bh** [reply]
Okay..I will check this..Thank you

**Reply by vishakha-vijay-bh** [reply]
@ad1992  After installing the package I cannot find exacalidraw-asset folder in dist. Can you please help me with this ?

**Reply by dwelle** [reply]
It should be in `node_modules/@excalidraw/excalidraw/dist`

**Reply by vishakha-vijay-bh** [reply]
hi team,
Loading chunk issue got resolved .but I am facing this error - Refused to load the font 'https://excalidraw.com/Cascadia.woff2' because it violates the following Content Security Policy directive: "font-src 'self' 'unsafe-inline' data: fonts.gstatic.com".

Can you please help me with this ?

**Reply by vishakha-vijay-bh** [reply]
hi,

Facing ChunkLoadError: Loading chunk 9 failed. intermittently. Can anyone help me with this ? 

Errors : 
ChunkLoadError: Loading chunk 9 failed.
Unexpected token '<'
ERROR ReferenceError: resolve is not defined

**Reply by ad1992** [reply]
@vishakha-vijay-bh are you using `window.EXCALIDRAW_ASSET_PATH` ? did you update the assets after upgrading excalidraw?

**Reply by vishakha-vijay-bh** [reply]
I have copied assets into different folder and given path of those assets.
